weasyprint/css/validation.py

```diff
@@ -685,6 +685,8 @@ def validate_content_token(base_url, token):
             style = args[-1]
             if style in ('none', 'decimal') or style in counters.STYLES:
                 return (name, args)
+        elif prototype in (('string', ['IDENT']),('string', ['IDENT', 'IDENT'])):
+            return (name, args)
```

Comment:
These keywords should also be case-insensitive. (But not the string/variable’s name!)